packages/api/src/external/carequality/shared.ts

```diff
@@ -179,3 +181,37 @@ export async function getCqAddress({
 export const cqOrgActiveSchema = z.object({
   active: z.boolean(),
 });
+
+export function cqLinkToPatientData(cqLink: CQLink): PatientData {
+  const patient = cqLink.patientResource;
+  const firstName = patient?.name.map(name => name.given).join(" ") ?? "";
+  const lastName = patient?.name.map(name => name.family).join(" ") ?? "";
+  const dob = patient?.birthDate ? buildDayjs(patient.birthDate).format(ISO_DATE) : "";
+  const genderAtBirth = patient?.gender === "male" ? "M" : patient?.gender === "female" ? "F" : "U";
```

Comment:
let's make a utility func for this so we don't have to rewrite this logic across the codebase